cli/src/cmd/new.rs

```diff
@@ -128,25 +132,26 @@ contract! {{
 
         /// Returns the current state.
         pub(external) fn get(&self) -> bool {{
+            println(&format!("Storage Value: {{:?}}", *self.value));
             *self.value
         }}
     }}
 }}
 
 #[cfg(test)]
 mod tests {{
-    use super::Flipper;
+    use super::{};
```

Comment:
maybe you mean to be using `use super::*;` here?

Reply:
That is probably more generic right? Can change.

Reply:
Resolved